services/kapacitor/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,73 @@
+kapacitor:
+  ## influxdb image version
+  ## ref: https://hub.docker.com/r/library/influxdb/tags/
+  ##
+  image:
+    repository: "kapacitor"
+    tag: "1.5.6-alpine"
```

Comment:
Same question here about whether the upstream Helm chart will handle this for us.

Reply:
See above.